www/doc/2.1/tutorial.html

```diff
@@ -93,6 +97,16 @@ <h2>Giving Herbie expressions</h2>
     tutorial let's move on to a more realistic example.
   </p>
 
+  <p>
+    Herbie measures accuracy by comparing a program's result against
+    the exact answer calculated using GNU MPFR. The difference
+    between these two is then measured in "bits of error" which
+    represents how many of the most significant bits that the
+    approximate and exact result agree on. This error is then
+    averaged across many different sample inputs to determine
+    the program's overall accuracy.
```

Comment:
There's an `error.html` page we should link to in here. It has more details so maybe this paragraph can be a bit shorter, maybe this:

> Herbie measures accuracy by comparing a program's result against
    the exact answer calculated using high-precision arithmetic. The difference
    between these two is then measured in "bits of error" which
    counts how many of the most significant bits that the
    approximate and exact result agree on. This error is then
    averaged across many different sample inputs to determine
    the program's overall accuracy.

Ok I didn't change much, this is a good paragraph.